cypress/support/page_objects/calculator.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Available operations enum
+ */
+export const Operations = {
+  Add: "0",
+  Subtract: "1",
+  Multiply: "2",
+  Divide: "3",
+  Concatenate: "4",
+};
+
+/**
+ * Calculator helper based on operations enum to validate results
+ */
+const calculator = {
+  [Operations.Add](num1, num2) {
+    return num1 + num2;
+  },
+  [Operations.Subtract](num1, num2) {
+    return num1 - num2;
+  },
+  [Operations.Multiply](num1, num2) {
+    return num1 * num2;
+  },
+  [Operations.Divide](num1, num2) {
+    return num1 / num2;
+  },
+  [Operations.Concatenate](num1, num2) {
+    return `${num1}${num2}`;
+  },
+};
+
```

Comment:
I prefer to have those utilities near all calculator related code, instead of importing it from another place. It's just easier to follow while the code base is small enough and those helpers are not shared between multiple test suites.